platform/integrations/slack_integration/setup.mdx

```diff
@@ -0,0 +1,163 @@
+---
+id: setup
```

Comment:
Can we plesae also small "About" page as a home for the slack integration covering the capabilities of the integration. It's goog for SEO, but also it's good when we want to link to it from somewhere, so we don't have to directly point to setup.